<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scutti</class>
 <widget class="QWidget" name="Scutti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="settingsPage">
      <widget class="QLabel" name="lblSettings">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>6</y>
         <width>126</width>
         <height>55</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Titillium Web</family>
         <pointsize>36</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblWidth">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>65</y>
         <width>50</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Width</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtWidth">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>92</y>
         <width>776</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rubik</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="lblHeight">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>176</y>
         <width>55</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Height</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtHeight">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>203</y>
         <width>776</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rubik</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="lblXStart">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>288</y>
         <width>58</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>X Start</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtXStart">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>315</y>
         <width>776</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rubik</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="lblYStart">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>399</y>
         <width>57</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Y Start</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtYStart">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>426</y>
         <width>776</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rubik</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="lblSnapKey">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>511</y>
         <width>102</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Capture Key</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtSnapKey">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>538</y>
         <width>776</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rubik</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="lblQuitKey">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>622</y>
         <width>74</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Quit Key</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtQuitKey">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>649</y>
         <width>776</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rubik</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="btnHomeNav">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>957</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Titillium Web</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblDelay">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>845</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Initial Delay</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblInterval">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>734</y>
         <width>63</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Interval</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtInterval">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>761</y>
         <width>776</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rubik</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtDelay">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>872</y>
         <width>776</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rubik</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="homePage">
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>801</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="16" column="0">
        <widget class="QPushButton" name="btnStart">
         <property name="font">
          <font>
           <family>Titillium Web</family>
           <pointsize>20</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>100</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Titillium Web</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Scutti</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblCollection">
         <property name="font">
          <font>
           <family>Titillium Web</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Collection Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>100</width>
           <height>350</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <widget class="QTextEdit" name="txtCollection">
         <property name="font">
          <font>
           <family>Titillium Web</family>
           <pointsize>24</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="btnSettingsNav">
         <property name="font">
          <font>
           <family>Titillium Web</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QComboBox" name="cmbSelect">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Titillium Web</family>
           <pointsize>20</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Manual Screenshot</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Automatic Screenshot</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Manual Camera</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Automatic Camera</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
